github/repos_test.go

```diff
@@ -629,7 +630,8 @@ func TestRepositoriesService_UpdateBranchProtection(t *testing.T) {
 					{Slug: String("tt"), ID: Int64(4)},
 				},
 			},
-			RequireCodeOwnerReviews: true,
+			RequireCodeOwnerReviews:      true,
+			RequiredApprovingReviewCount: 0,
```

Comment:
Let's give it a value in the valid 1-6 range, since it's documented that way:

> Use a number between 1 and 6.

Reply:
If the value hasn't been set, by default, it comes back as 0. I'll update the documents.